content/zh-cn/docs/concepts/services-networking/ingress.md

```diff
@@ -318,7 +318,7 @@ Ingress 中的每个路径都需要有对应的路径类型（Path Type）。未
 
 * `Prefix`：基于以 `/` 分隔的 URL 路径前缀匹配。匹配区分大小写，并且对路径中的元素逐个完成。
   路径元素指的是由 `/` 分隔符分隔的路径中的标签列表。
-  如果每个 _p_ 都是请求路径 _p_ 的元素前缀，则请求与路径 _p_ 匹配。
+  如果每个 **p** 都是请求路径 **p** 的元素前缀，则请求与路径 **p** 匹配。
```

Comment:
It is fine to use _italics_ for English text, e.g. _p_.
We change italics to bold texts for Chinese characters, because _斜体中文_ is ugly than **粗体中文**.